src/Sylius/Bundle/ShippingBundle/Resources/config/services.xml

```diff
@@ -67,12 +70,18 @@
             <argument>shipping rule checker</argument>
         </service>
 
+        <service id="sylius.registry.shipping_methods_resolver" class="%sylius.registry.shipping_methods_resolver.class%">
+            <argument>%sylius.shipping_methods_resolver.interface%</argument>
+            <argument>Shipping methods resolver</argument>
```

Comment:
According to [service implementation](https://github.com/Sylius/Sylius/blob/3e0fd8522683bc6c893cc583fdb9d543b56f96c2/src/Sylius/Component/Registry/PrioritizedServiceRegistry.php#L35) ``$context``parameter should be "human readable".